src/sentry/integrations/jira/client.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+import datetime
+import jwt
+from six.moves.urllib.parse import urlparse
+
+from sentry.http import build_session
+from sentry.utils.http import absolute_uri
+
+from .utils import get_query_hash
+
+JIRA_KEY = '%s.jira' % (urlparse(absolute_uri()).hostname, )
+
+
+class JiraApiClient(object):
+    ISSUE_URL = '/rest/api/2/issue/%s'
+
+    def __init__(self, base_url, shared_secret):
+        self.base_url = base_url
+        self.shared_secret = shared_secret
+
+    def request(self, method, path, data=None, params=None, headers=None):
```

Comment:
the old version had even more named arguments, so i'm just going to leave them. i will add **kwargs though since i think eventually some of this will be moved into a shared class and presumably other integrations might need other arguments